shared/translations/languages/es.json

```diff
@@ -1569,5 +1569,11 @@
     "Wallet created!": "Cartera creada!",
     "Access decentralized finance with Bitcoin-grade security, strength and immutability.": "Accede a finanzas descentralizadas con el grado de seguridad, fuerza e inmutabilidad de Bitcoin.",
     "Continue": "Continúa"
+  },
+  "screens/ReceiveDTokenScreen": {
+    "Please note the MINIMUM deposit amount is {{MinimumBtcAmount}}!": "Please note the MINIMUM deposit amount is {{MinimumBtcAmount}}!",
```

Comment:
other languages like fr and it don't have this line

Reply:
just stubs, forwarded to translation team